rsl-learning2022-Risheel/view1/view/app/src/main/res/layout/activity_image_view.xml

```diff
@@ -0,0 +1,126 @@
+                android:layout_height="200dp"
+                android:src="@drawable/tulips"
+                android:scaleType="centerInside"
+                android:adjustViewBounds="true"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textSize="30dp"
+                android:textColor="@color/teal_200"
+                android:text="center  inside "
+                />
+            <ImageView
+                android:layout_width="400dp"
+                android:layout_height="400dp"
+                android:src="@drawable/tulips"
+                android:scaleType="fitXY"
+                android:adjustViewBounds="true"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textSize="30dp"
+                android:textColor="@color/teal_200"
+                android:text="fitxy images"
+                />
+            <ImageView
+                android:layout_width="400dp"
+                android:layout_height="400dp"
+                android:src="@drawable/vase"
+                android:scaleType="centerInside"
+                android:adjustViewBounds="true"
+                app:tint="@color/tint_color"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textSize="30dp"
+                android:textColor="@color/teal_200"
+                android:text="center inside with images tint"
+                />
+            <ImageView
+                android:layout_width="400dp"
+                android:layout_height="400dp"
+                android:src="@drawable/vase"
+                android:scaleType="centerInside"
+                android:adjustViewBounds="true"
+                app:tint="@color/tint_transparency"
+                android:tintMode="screen"/>
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:textSize="30dp"
+                android:textColor="@color/teal_200"
+                android:text="tint with screen mode"
+                />
+            <ImageView
+                android:layout_width="400dp"
+                android:layout_height="400dp"
+                android:src="@drawable/vase"
+                android:scaleType="centerInside"
+                android:adjustViewBounds="true"
+                app:tint="@color/tint_transparency"
+                android:tintMode="src_in"/>
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="30dp"
+                android:text="tint with src in mode"
+                android:textColor="@color/teal_200"
+                android:textSize="30dp" />
+        </LinearLayout>
```

Comment:
ImageViews missing the following: 

- Use of additional scaleType values.
- Use of fully transparent image.
- Correct use of tint and tintMode for images with no transparency and some transparency.